homeassistant/components/zha/light.py

```diff
@@ -157,6 +153,7 @@ def async_set_state(self, state):
         if self._level_channel:
             await self.async_accept_signal(
                 self._level_channel, SIGNAL_SET_LEVEL, self.set_level)
+        async_track_time_interval(self.hass, self.async_update, SCAN_INTERVAL)
```

Comment:
`async_update` doesn't accept any arguments. `async_track_time_interval` wants to pass a time argument to the action function.

Reply:
Oy, ok.. 1 sec

Reply:
fixed